### ISSUE TITLE
Exceptions thrown in development aren't logged to the console

### ISSUE THREAD

**Issue by paulcwatts** [Exelord/ember-rollbar-client#18]
I think this might be related to https://github.com/Exelord/ember-rollbar-client/issues/14. This seems to be a problem with the instance-initializer: the initializer *always* sets `Ember.onerror`, even if rollbar is disabled. However, by default, `Ember.onerror` is `undefined`, which causes the overridden `Ember.onerror` to do nothing, thereby squelching the exception.

This can be resolved by only setting `Ember.onerror` if the rollbar service is enabled:

```javascript
export function initialize(appInstance) {
  let rollbarService = appInstance.lookup('service:rollbar');
  if (rollbarService.get('enabled')) {
    let oldOnError = Ember.onerror || function() {};

    Ember.onerror = (...args) => {
      oldOnError(...args);
      rollbarService.error(...args);
    };
  }
}
``` 

However, that might not work if you want the `enabled` state to change after the app boots. (I'm not sure if that's something you want.)

Alternatively, you can add an `else` case in `onerror` to rethrow the error if rollbar is disabled, something like:

```javascript
  Ember.onerror = (...args) => {
    oldOnError(...args);

    if (rollbarService.get('enabled')) {
      rollbarService.error(...args);
    } else {
      throw args[0];
    }
  };
```

I'd be happy to create a PR, but I don't know the best way you'd like to proceed.

**Comment by paulcwatts** [comment]
Just FYI, this is using version 0.5.1.

**Comment by Exelord** [comment]
Thanks for reporting. I will look at closer this weekend :)

**Comment by paulcwatts** [comment]
I took a shot at fixing this https://github.com/Exelord/ember-rollbar-client/pull/19

**Comment by sdhull** [comment]
Yeah FWIW, wrt #14, errors stopped being logged to the console after some time. No idea why, so hopefully this fixes it 🤞

**Comment by Exelord** [comment]
This issue should be fixed in `v0.5.1`. Feel free to open it.
